doc/ChangeLog

```diff
@@ -52,22 +52,22 @@ Notes of particular relevance for users
 
 Caveats for users (e.g., need to interpolate initial conditions): none
 
-Changes to CTSM's user interface (e.g., new/renamed XML or namelist variables):
+Changes to CTSM's user interface (e.g., new/renamed XML or namelist variables): none
```

Comment:
Here and elsewhere: I prefer if things are explicitly answered rather than left blank